paths/job_locales/complete_review.yaml

```diff
@@ -0,0 +1,59 @@
+---
+summary: Review a job locale
+description: Marked Job locale as reviewed.
```

Comment:
I'd change it to `Mark job locale as reviewed`

Reply:
👍  changed, thx